src/test/java/BuyShirt.java

```diff
@@ -0,0 +1,76 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxBinary;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import static java.lang.Thread.sleep;
+
+public class BuyShirt {
+
+    private WebDriver driver;
+
+    @BeforeEach
+    public void before() {
+
+        WebDriverManager.chromedriver().setup();
+
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--headless");
+        driver = new ChromeDriver(options);
+
+    }
+
+    @Test
+    public void webdriverChrome() throws InterruptedException {
+
+        driver.get("http://automationpractice.com/");
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("#block_top_menu > ul > li:nth-child(3) > a"))).click();
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("#center_column a.button.ajax_add_to_cart_button.btn.btn-default"))).click();
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("[style*=\"display: block;\"] .button-container > a"))).click();
+
+        new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+
+            WebElement emailUser = (WebElement) new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("#email")));
+            emailUser.click();
+
+            WebElement password = (WebElement) new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("#passwd")));
+            password.click();
+
+            emailUser.sendKeys("aperdomobo@gmail.com");
+            password.sendKeys("WorkshopProtractor");
+
+            WebElement signIn = (WebElement) new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("#SubmitLogin")));
+            signIn.click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".checker"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector("a.bankwire"))).click();
+
+            new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable(By.cssSelector(".cart_navigation span"))).click();
```

Comment:
fix the indentation

Reply:
Done

Reply:
First question; It is possible to have only one WebDriverWail for all actions?  So you don't have to instantiate that class every time?

Reply:
**Second question: `**_new WebDriverWait(driver, 20).until(ExpectedConditions.elementToBeClickable..._`  can we do this only for 1 element on the page (the slowest one or something like that) so you don't have to put the wait for all elements?

Reply:
Ambas propuestas son muy vailidas Yeli, sin embargo el objetivo de esta tarea pienso que es que pudiera automatizar un caso, no necesariamente con practicas

Creo que la proxima tarea debería tener en cuenta estas 2 cosas que dices